vultr/resource_startup_script.go

```diff
@@ -67,6 +70,12 @@ func resourceStartupScriptRead(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("Error getting startup script (%s): %v", d.Id(), err)
 	}
 
+	if (lib.StartupScript{}) == script {
```

Comment:
Nit: could you reverse the order of these operands? I think it’s geberally easier to read if the subject of the comparison, in this case `script`, comes first